common/static/common/js/discussion/discussion_router.js

```diff
@@ -0,0 +1,167 @@
+/* globals DiscussionThreadListView, DiscussionThreadView, DiscussionUtil, NewPostView */
```

Comment:
Many of these files have a `/* globals var1, var2, var3 */` at the top, which looks like [it targets JSHint](http://jshint.com/docs/#inline-configuration). If we switch `globals` to `global` this will work in ESLint as well (see [ESLint `no-undef` docs](http://eslint.org/docs/rules/no-undef))